source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -513,12 +513,21 @@ public TService CreateClient<TService>(Action<ServiceEndPoint> modifyServiceEndp
 
             public TClientService CreateClient<TService, TClientService>(Action<ServiceEndPoint> modifyServiceEndpoint)
             {
-                var serviceEndpoint = new ServiceEndPoint(ServiceUri, thumbprint, proxyDetails);
+                var serviceEndpoint = ServiceEndpoint();
                 modifyServiceEndpoint(serviceEndpoint);
                 return new AdaptToSyncOrAsyncTestCase().Adapt<TService, TClientService>(forceClientProxyType, Client, serviceEndpoint);
             }
+
+            public ServiceEndPoint ServiceEndpoint()
```

Comment:
nit: replace line:533 `var serviceEndpoint = new ServiceEndPoint(ServiceUri, thumbprint, proxyDetails);` with a call to this new method

Reply:
Ignore. I ended up doing this in another PR as a test needed access to the ServiceEndpoint